paper_sections/02-gams.Rmd

```diff
@@ -89,19 +108,19 @@ par(mar=c(0, 0, 0, 0) + 0.1)
 vis.gam(b, view=c("s.depth","salinity"), theta=-60, phi=30, color="bw")
 ```
 
-We represent the terms in our model as basis functions, which end up as additional columns in our design matrix and parameter vector, and penalties, which penalize the likelihood and stop our model from being too wiggly. Taking a pragmatic Bayesian approach to the problem, the penalty is really a prior on how we think the model should act. In which case the penalty matrix itself is a prior precision matrix (inverse variance) for the term. With that in mind, we can think about random effects as "smooths" in our model, albeit ones with ridge penalies [@kimeldorf_correspondence_1970; @wood_generalized_2017]. For instance, to include a random effect modelling between group variation in intercepts there will be one basis function for each level of the grouping variable, that takes a value of 1 for any observation in that group and 0 for any observation not in the group. The penalty matrix for these terms is a $n_g$ by $n_g$ identity matrix, where $n_g$ is the number of groups. This means that each group-level coefficient will be penalized in proportion to its squared deviation from zero. This is equivilent to how random effects are estimated in standard mixed effect models. The penalty term here is proportionate to the inverse of the variance of the fixed effect estimated by standard hierarchical model solvers [@verbyla_analysis_2002  does this contradict what's above??]. 
+**GLS: The first sentence below is quite repetitive. That Bayesian bit comes out of nowhere here, even with the repetitive intro. Likewise the bit above about the interactions seems odd, and the analogy of `A*B` == `te(A, B)` vs `A + B + A:B` == `ti(A) + ti(B) + ti(A, B)` is not right; `A*B` is a syntactic shortcut for the marginal effects plus the interaction --- it decomposes into 3 separate terms unlike `te()`, and all the `ti()`s lead to extra penalties over the `te()` version.**
```

Comment:
will have a hack at rewriting this -- I think the Bayesian bit I wrote while I was thinking about Bayesian interpretations, so had that on my mind...